.github/workflows/pytest-dbutils-sqlitefuncts.yml

```diff
@@ -0,0 +1,55 @@
+---
+# This workflow is meant as a foundational workflow for running integration/unit tests on the
+# platform.
+# This workflow also shows the caching mechanisms available for storage
+# and retrieval of cache for quicker setup of test environments.
+
+
+name: Testing sqlitefuncts
+on: [pull_request]
+
+jobs:
+  build:
+    env:
+      TEST_FILE: sqlitefuncts.py
```

Comment:
I think this should be the whole filename....you will confuse people (me included) without the full file.